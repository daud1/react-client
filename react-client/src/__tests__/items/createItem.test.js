import React from 'react';
import {shallow, render, mount } from 'enzyme';
import CreateItem from '../../components/items/createItem.component';
import shallowToJson from 'enzyme-to-json';

describe ('CreateItem Form Component', () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<CreateItem />);
        expect(wrapper.exists(<div></div>)).toBe(true);
    });

    it('matches snapshot', () => {
        const wrapper = shallow(<CreateItem />);
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
})