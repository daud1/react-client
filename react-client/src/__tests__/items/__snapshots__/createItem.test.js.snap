// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateItem Form Component matches snapshot 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <TextField
      disabled={false}
      floatingLabelFixed={false}
      floatingLabelText="Enter item name"
      fullWidth={false}
      multiLine={false}
      name="name"
      rows={1}
      type="text"
      underlineShow={true}
    />
    <button
      className="btn btn-primary"
      type="submit"
    >
      Add Item
    </button>
  </form>
</div>
`;
